daemon/inertiad/auth/users_test.go

```diff
@@ -26,11 +26,11 @@ func TestAddUserAndIsCorrectCredentials(t *testing.T) {
 	err = manager.AddUser("bobheadxi", "best_person_ever", true)
 	assert.Nil(t, err)
 
-	_, correct, err := manager.IsCorrectCredentials("bobheadxi", "not_quite_best")
+	_, correct, _, err := manager.IsCorrectCredentials("bobheadxi", "not_quite_best", "")
 	assert.Nil(t, err)
 	assert.False(t, correct)
 
-	_, correct, err = manager.IsCorrectCredentials("bobheadxi", "best_person_ever")
+	_, correct, _, err = manager.IsCorrectCredentials("bobheadxi", "best_person_ever", "")
```

Comment:
is there any way we could reduce the number of returned values in `IsCorrectCredentials`? 🤔

Reply:
Agreed, that function is very unwieldy right now.

Probably better to split totp checking into its own function.